Stdsub

jospoortvliet opened the pull request:
This are changes to the website due to changes in ownCloud org and Inc. as well a a number of general improvements. @tomneedham could you give this a look? Some more FAQ items might be coming, as well as some refinements to wording on the new Support page, but overall - this is getting pretty complete. This should, ideally, go live Monday midnight or Tuesday morning.

tomneedham commented:
The teaser image is really poor quality and doesn't show ownCloud 8. Not sure on the mobile app, but the current Android version looks quite different.

tomneedham commented on page-homepage.php:
```diff
@@ -221,32 +221,35 @@ function() {
 					<a class="btn btn-default btn-lg hostingbtn" href="/providers/"><i class="icon-search" ></i> Find hosting</br><small>with a third party</small></a>
 				</div>
-				<a class="btn btn-default btn-lg client-install" href="/install/#desktop"><i class="icon-mobile" ></i> Install Clients</br><small>for desktop and mobile</small></a>
+				<br /><br /><a class="btn btn-default btn-lg client-install" href="/install/#desktop"><i class="icon-mobile" ></i> Install Clients</br><small>for desktop and mobile</small></a>
```
Is there a reason why we want this bellow, uses up a fair bit of space?

jospoortvliet replied:
I think it looks better... Just that. But maybe it looks different in different browsers...
![eight1](https://cloud.githubusercontent.com/assets/551757/6561650/35935f4c-c693-11e4-9302-756dcbd92937.png)
is what it looks like here. I can either have the three buttons against each other (which I find ugly) or have this (indeed rather big) empty space. least bad, imho.

tomneedham commented on page-faq.php:
```diff
@@ -210,32 +224,43 @@
+</ul>
 <a name="entcomparison"></a>
-<h3>How do the Community and Enterprise compare?</h3>
+<h3>What are the Standard Subscription and Enterprise Subscription?</h3>
```
* What is

jospoortvliet replied:
ack

tomneedham commented on page-install.php:
```diff
@@ -9,8 +9,9 @@
-    <p>Looking for <a href="/changelog">older versions</a>?<br />
-    Want to use <a target="_blank" href="https://owncloud.com/products/enterprise/">ownCloud for the Enterprise?</a></p>
+    <p>Looking for <a href="/changelog">older versions</a>?
+<!--     <br /> Want to use <a target="_blank" href="https://owncloud.org/support">ownCloud professionally?</a> -->
```
Let's just remove this, we have git so it's always in the history. Otherwise we end up with all this lying around.

jospoortvliet replied:
ack

jospoortvliet commented:
About the teaser image. I've changed it, it now shows oC 8. In general, it is tiny, but I think it looks nicer to show at least an app and the web UI rather than only the web UI, that is why I replaced it. I'd need the source from Aachim (or a higher res version) to make it any better than this...

The phone shows the iOS app, as far as I can tell.

Other changes implemented. What do you think, @tomneedham ?

tomneedham commented:
@jospoortvliet Let's continue reviewing on staging
